### DISCUSSION TITLE
Welcome to TabTabTab Discussions!

### DISCUSSION THREAD

**Discussion by okaryo** [okaryo/TabTabTab#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by FreePDF64** [comment]
Hi,
I'm an IT infrastructure Manager by profession. I'm working daily with my Edge Browser and many many tabs. For this I need a perfect Tab Manager - like TabTabTab.

I would like to contribute with my ideas to make this AddOn almost perfect for working with tabs and tab groups.

KR,
Michael

**Reply by okaryo** [reply]
Hi Michael,

Thank you so much for your contribution! I'm looking forward to improving TabTabTab together! Feel free to use Discussions to share any ideas you have!

**Comment by FreePDF64** [comment]
My next wish for TabTabTab is:
I would like to see the number of tabs in a tab group displayed.

Thank you for quick implementation - if this is ok for you or the rest of the community here?! Or you plan to implement this in another way ;)

**Reply by okaryo** [reply]
Thanks for your suggestion. I already have plans for that feature and it's been issued in #40 . I'm currently working on syncing the open/close state of groups in #87 , and I plan to address this feature while I'm at it.

By the way, how about we use the `ideas` category in Discussions for each feature conversation? Apologies if we're still figuring out the best way to use Discussions. I hope we can find a good way to operate this together.
https://github.com/okaryo/TabTabTab/discussions/categories/ideas

**Reply by FreePDF64** [reply]
That sounds good. How can we move my Post to the ideas category. Can you do this or do I delete my Post here and write it again under /ideas?

**Reply by okaryo** [reply]
It seems that I cannot move your post. Could you please create a new Discussion in the `ideas` category?

**Reply by okaryo** [reply]
There's no need to delete your post, so don't worry! Thank you.

**Reply by FreePDF64** [reply]
Done... ;)
